src/modules/cards/cards.module.scss

```diff
@@ -1,24 +1,25 @@
-@import 'src/assets/stylesheets/base_styles/typography';
-@import 'src/assets/stylesheets/base_styles/colors';
-@import 'src/assets/stylesheets/utilities/functions';
-@import 'src/assets/stylesheets/utilities/mixins';
+@import 'src/assets/stylesheets/global';
```

Comment:
@drewdrewthis yeah I had that thought in the back of my mind as well...I'm going to be investigating more into post css today, so we should have a better idea